packages/webpack-cli/README.md

```diff
@@ -58,6 +58,932 @@ Options
+  --no-stats-exclude-assets-reset                                                    Negates stats-exclude-assets-reset
+  --no-stats-exclude-modules-reset                                                   Negates stats-exclude-modules-reset
+  --no-stats-hash                                                                    Negates stats-hash
+  --no-stats-ids                                                                     Negates stats-ids
+  --no-stats-logging-debug-reset                                                     Negates stats-logging-debug-reset
+  --no-stats-logging-trace                                                           Negates stats-logging-trace
+  --no-stats-module-assets                                                           Negates stats-module-assets
+  --no-stats-module-trace                                                            Negates stats-module-trace
+  --no-stats-modules                                                                 Negates stats-modules
+  --no-stats-nested-modules                                                          Negates stats-nested-modules
+  --no-stats-optimization-bailout                                                    Negates stats-optimization-bailout
+  --no-stats-orphan-modules                                                          Negates stats-orphan-modules
+  --no-stats-output-path                                                             Negates stats-output-path
+  --no-stats-performance                                                             Negates stats-performance
+  --no-stats-provided-exports                                                        Negates stats-provided-exports
+  --no-stats-public-path                                                             Negates stats-public-path
+  --no-stats-reasons                                                                 Negates stats-reasons
+  --no-stats-runtime                                                                 Negates stats-runtime
+  --no-stats-source                                                                  Negates stats-source
+  --no-stats-timings                                                                 Negates stats-timings
+  --no-stats-used-exports                                                            Negates stats-used-exports
+  --no-stats-version                                                                 Negates stats-version
+  --no-stats-warnings                                                                Negates stats-warnings
+  --no-stats-warnings-filter-reset                                                   Negates stats-warnings-filter-reset
+  --no-watch-options-ignored-reset                                                   Negates watch-options-ignored-reset
+  --no-watch-options-stdin                                                           Negates watch-options-stdin
```

Comment:
Let's use

### webpack 4

```
...
```

### webpack 5

```
...
````

Also we have very many spaces between flags and description. Can we fix it?

Reply:
there were many spaces because some flags have a very very long name like 
`--optimization-split-chunks-fallback-cache-group-automatic-name-delimiter`
`--optimization-split-chunks-fallback-cache-group-max-initial-size.` 

I managed to reduce down spaces to some extent.

Reply:
I will push my changes soon.